cmd/rpc.go

```diff
@@ -73,7 +82,12 @@ func InitClient(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
-	client, err := rpc.NewClient(cmd.Context(), requestURL, authTokenFlag)
+	var opts []rpc.RPCClientOption
+	if timeoutFlag > 0 {
+		opts = append(opts, rpc.WithTimeout(timeoutFlag))
+	}
+
+	client, err := rpc.NewClient(cmd.Context(), requestURL, authTokenFlag, opts...)
```

Comment:
There is a simpler way to achieve the same result.

We can pass through the timeout in the context. The timeout is respected by the RPC HTTP client performing the request, which avoids creating a new option for the client.